package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@folio/data-import",
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
These changes are under v7.0.0 in CHANGELOG, is this needed to increase the version here?

Reply:
I'll undo this, thanks